### ISSUE TITLE
Temporary admins can not see the admin chat.

### ISSUE THREAD

**Issue by OttoRocket-LVP** [LVPlayground/playground#744]
Temporary admins can not see the admin chat.
There messages are also shown as a report instead of temp admin.

**Comment by OttoRocket-LVP** [comment]
reports are not showing either

**Comment by Specifer** [comment]
There is a striking thing that the **original player state still stays**. I tested with lithirm in staging, My original account was as management level. with lith's help, I took my admin and he made me a temp. But when I admin chatted, It referred me as manager. but when I was unregistered temp. I could see Admin notice but not been able to chat but sent it as a message from player. And couldn't gang chat in another gang.

**Comment by RussellLVP** [comment]
Yeah. I landed a tentative fix a couple of days ago that's now live on the staging server. Would appreciate someone testing this :)

**Comment by Specifer** [comment]
Cool, will do!

**Comment by Specifer** [comment]
@RussellLVP, The problem got worse. Even as an admin or manager. I can't use admin chat or use kickflip, jetpack, fly, lvp access lvp settings shows the result of only /lvp (it is the server which I'm playing on). But /vr, /fix works
